cypress/e2e/shared/checks.test.ts

```diff
@@ -537,8 +545,96 @@ describe('Checks', () => {
         // Need to trigger mouseout else the popover obscures the other buttons
         .trigger('mouseout')
 
+      // implicitly verify id value while setting up intercept
+      cy.getByTestID('copy-resource-id').then(elem => {
+        const checkID = elem
+          .text()
+          .replace(/^\D+/g, '')
+          .substring(0, 16)
+        return cy
+          .intercept('PATCH', `/api/v2/checks/${checkID}*`)
+          .as('patchCheck')
```

Comment:
I think you already declared a `patchCheck` intercept above on line 527 right?

Reply:
Yes, but the declaration on line 527 was a mistake.  It has been removed.